environments/auth/package.json

```diff
@@ -3,8 +3,8 @@
   "name": "environments-auth",
   "version": "0.0.1",
   "scripts": {
-    "pull": "../pull-environments.sh -r us-east-1",
-    "pull:phone": "yarn pull -i \"\\./auth-with-((phone-and-sms-mfa)|(phone-number)|(totp-and-sms-mfa))\"",
-    "pull:non-phone": "yarn pull -e \"\\./auth-with-((phone-and-sms-mfa)|(phone-number)|(totp-and-sms-mfa))\""
+    "pull": "../pull-environments.sh",
+    "pull:phone": "yarn pull -r us-east-1 -i \"\\./auth-with-((phone-and-sms-mfa)|(phone-number)|(totp-and-sms-mfa))\"",
```

Comment:
why do we have `phone` in `us-east-1`, but `non-phone` in `us-east-2`?

Reply:
I'll answer that async!